en/developer_guide/extend_document.rst

```diff
@@ -0,0 +1,60 @@
+Extend document and repository
+==============================
+
+Use case
+----------
+
+OpenOrchestra provides users and developpers with a set of entities, for example the content entity, functionnal kernel of a CMS.
```

Comment:
I would rather say :+1: 

Open Orchestra presents a list of model object to the integrators. To customize those objects, it could be necessary to extend the implemented documents and repositories.